media-libs/flac/flac-1.5.0-r1.ebuild

```diff
@@ -41,6 +41,7 @@ multilib_src_configure() {
 	local myeconfargs=(
 		--disable-doxygen-docs
 		--disable-examples
+		--disable-oss-fuzzers # clang only
```

Comment:
Well, more that we don't care for fuzzing as part of the ebuild anyway. Is it on by default, or what?